website/docs/migration-from-import.md

```diff
@@ -15,10 +15,12 @@ const typeDefs = importSchema(join(__dirname, 'schema.graphql'));
 
 After;
 ```ts
-import { loadSchemasSync } from '@graphql-tools/load';
-import { GraphQLFileLoader } from '@graphql-tools/code-file-loader';
+import { loadSchema } from '@graphql-tools/load';
+import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader';
 
-const typeDefs = loadSchemasSync(join(__dirname, 'schema.graphql'));
+const typeDefs = await loadSchema(join(__dirname, 'schema.graphql'), {
```

Comment:
I caught this two. The result is not typeDefs but schema. Schema cannot be passed to `makeExecutableSchema({ typeDefs: schema })` for example.

Reply:
@ardatan Can probably help us out here